docs/UserGuide.adoc

```diff
@@ -297,9 +293,9 @@ image::demo/Search.png[]
 
 * *View help*: `help`
 * *Exit the program*: `exit`
-* *Create a new note*: `new -t TITLE [-b BODY] [-o]`
+* *Create a new note*: `new -t TITLE [-o]`
 * *Open an existing note*: `open [-t] [PATH/TO/]TITLE`
-* *Edit an existing note*: `edit -b BODY`
+* *Edit an existing note*: `edit`
```

Comment:
Should we rephrase this as "Opens an editing window"?

Reply:
Ok!